File: app/src/main/java/com/rdev/tryp/model/favourite_driver/DriversItem.java
package com.rdev.tryp.model.favourite_driver;

import com.google.gson.annotations.SerializedName;

public class DriversItem {

    @SerializedName("max_luggage")
    private int maxLuggage;

    @SerializedName("fare")
    private double fare;

    @SerializedName("max_passenger")
    private int maxPassenger;

    @SerializedName("driver")
    private Driver driver;

    @SerializedName("category")
    private String category;

    @SerializedName("type")
    private String type;

    @SerializedName("vehicle")
    private Vehicle vehicle;

    public void setMaxLuggage(int maxLuggage) {
        this.maxLuggage = maxLuggage;
    }

    public int getMaxLuggage() {
        return maxLuggage;
    }

    public void setFare(double fare) {
        this.fare = fare;
    }

    public double getFare() {
        return fare;
    }

    public void setMaxPassenger(int maxPassenger) {
        this.maxPassenger = maxPassenger;
    }

    public int getMaxPassenger() {
        return maxPassenger;
    }

    public void setDriver(Driver driver) {
        this.driver = driver;
    }

    public Driver getDriver() {
        return driver;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getCategory() {
        return category;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }

    public void setVehicle(Vehicle vehicle) {
        this.vehicle = vehicle;
    }

    public Vehicle getVehicle() {
        return vehicle;
    }

    @Override
    public String toString() {
        return
                "DriversItem{" +
                        "max_luggage = '" + maxLuggage + '\'' +
                        ",fare = '" + fare + '\'' +
                        ",max_passenger = '" + maxPassenger + '\'' +
                        ",driver = '" + driver + '\'' +
                        ",category = '" + category + '\'' +
                        ",type = '" + type + '\'' +
                        ",vehicle = '" + vehicle + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/DriversItem.kt
package com.rdev.tryp.model

import com.google.gson.annotations.SerializedName


class DriversItem {

    @SerializedName("max_luggage")
    var maxLuggage: Int = 0

    @SerializedName("fare")
    var fare: Double = 0.toDouble()

    @SerializedName("max_passenger")
    var maxPassenger: Int = 0

    @SerializedName("driver")
    var driver: Driver? = null

    @SerializedName("location")
    var location: Location? = null

    @SerializedName("category")
    var category: String? = null

    @SerializedName("type")
    var type: String? = null

    @SerializedName("vehicle")
    var vehicle: Vehicle? = null

    override fun toString(): String {
        return "DriversItem{" +
                "max_luggage = '" + maxLuggage + '\''.toString() +
                ",fare = '" + fare + '\''.toString() +
                ",max_passenger = '" + maxPassenger + '\''.toString() +
                ",driver = '" + driver + '\''.toString() +
                ",location = '" + location + '\''.toString() +
                ",category = '" + category + '\''.toString() +
                ",type = '" + type + '\''.toString() +
                ",vehicle = '" + vehicle + '\''.toString() +
                "}"
    }

}